itest/assets_test.go

```diff
@@ -825,19 +908,103 @@ func createAssetInvoice(t *testing.T, dstRfqPeer, dst *HarnessNode,
 	rate, err := rfqrpc.UnmarshalFixedPoint(rpcRate)
 	require.NoError(t, err)
 
+	t.Logf("Got quote for %v asset units per BTC", rate)
+
 	assetUnits := rfqmath.NewBigIntFixedPoint(assetAmount, 0)
 	numMSats := rfqmath.UnitsToMilliSatoshi(assetUnits, *rate)
-	mSatPerUnit := uint64(decodedInvoice.NumMsat) / assetAmount
+	mSatPerUnit := float64(decodedInvoice.NumMsat) / float64(assetAmount)
 
 	require.EqualValues(t, numMSats, decodedInvoice.NumMsat)
 
-	t.Logf("Got quote for %d sats at %v msat/unit from peer %x with SCID "+
-		"%d", decodedInvoice.NumMsat, mSatPerUnit, dstRfqPeer.PubKey[:],
-		resp.AcceptedBuyQuote.Scid)
+	t.Logf("Got quote for %d mSats at %3f msat/unit from peer %x with "+
+		"SCID %d", decodedInvoice.NumMsat, mSatPerUnit,
+		dstRfqPeer.PubKey[:], resp.AcceptedBuyQuote.Scid)
 
 	return resp.InvoiceResult
 }
 
+// assertInvoiceHtlcAssets makes sure the invoice with the given hash shows the
+// individual HTLCs that arrived for it and that they show the correct asset
+// amounts when decoded.
+func assertInvoiceHtlcAssets(t *testing.T, node *HarnessNode,
```

Comment:
I don't think this function is used yet in the codebase. I think we should add this to the PR where its used.

Reply:
I think this came from your other PR. It's also in your other PR.